guides/files_experimental.md

```diff
@@ -0,0 +1,148 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'files-guide'
+is: 'experimental'
+---
+
+
+# Files in IBM Watson Work Services
+
+**Note: This documentation and underlying APIs and structures ARE subject to change**
```

Comment:
why do we need to state this - `EXPERIMENTAL` category is for this so can we just remove?